conf/test.config

```diff
@@ -0,0 +1,8 @@
+params {
```

Comment:
Created a test profile so that the test command is short: `./main.nf -profile docker,local`